crates/actor/src/compat/wasmcloud/bus/lattice.rs

```diff
@@ -1,95 +1,27 @@
-use std::sync::RwLock;
+use std::sync::{OnceLock, RwLock};
 
-use once_cell::sync::Lazy;
+/// The default link name in use by any target
+const DEFAULT_LINK_NAME: &str = "default";
 
-pub(crate) const WASI_BLOBSTORE_BLOBSTORE_TARGET: Lazy<RwLock<Option<TargetEntity>>> =
-    Lazy::new(RwLock::default);
-pub(crate) const WASI_KEYVALUE_ATOMIC_TARGET: Lazy<RwLock<Option<TargetEntity>>> =
-    Lazy::new(RwLock::default);
-pub(crate) const WASI_KEYVALUE_EVENTUAL_TARGET: Lazy<RwLock<Option<TargetEntity>>> =
-    Lazy::new(RwLock::default);
-pub(crate) const WASI_LOGGING_LOGGING_TARGET: Lazy<RwLock<Option<TargetEntity>>> =
-    Lazy::new(RwLock::default);
-pub(crate) const WASMCLOUD_MESSAGING_CONSUMER_TARGET: Lazy<RwLock<Option<TargetEntity>>> =
-    Lazy::new(RwLock::default);
+/// Any string that represents a link between two components on the lattice,
+/// possibly a ComponentID or or a target group
+type LinkName = String;
 
-/// Actor identifer
-#[derive(Clone, Eq, PartialEq)]
-pub enum ActorIdentifier {
-    /// Actor public key
-    PublicKey(String),
-    /// Actor call alias
-    Alias(String),
-}
-
-/// Target entity
-#[derive(Clone, Eq, PartialEq)]
-pub enum TargetEntity {
-    /// Link target paired with an optional name
-    Link(Option<String>),
-    /// Actor target
-    Actor(ActorIdentifier),
-}
+/// The current link name in use
+pub(crate) const LINK_NAME: OnceLock<RwLock<LinkName>> = OnceLock::new();
```

Comment:
I'm a little worried about this (granted, this is the beginning of my review so I might be missing something.) Is this a global link name? As in, two different actors will modify the same link name here in the OnceLock?

I think we definitely want an "in use" link name per component instance

Reply:
See earlier note about `compat` -- shouldn't it be gone for 1.0?

Reply:
Removed `compat` -- probably going to increase the change set quite a bit here to remove the support for all non-components, but we'd need to do this anyway